SummerProject/article_db/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from ..user_auth.models import User
+
+from enum import Enum
+
+class SocialNetworkChoice(Enum):
+    FB = "Facebook"
+    TR = "Twitter"
+    GGL = "Google"
+
+class Category(models.Model):
```

Comment:
Lets create a separate app for categories and move the `Category` and `Team` models to the new app.